packages/extension-browser/src/devtools/views/app.tsx

```diff
@@ -30,12 +32,20 @@ export type Props = {
     results?: ResultsData;
 };
 
+/*
+ * If we need to show the notification, we don't want it to be removed
+ * from the dom after the status change so we don't lose the animation.
+ */
+const addNotification = showOptIn();
```

Comment:
Since this never changes if it's `true` when we start, why not always leave the notification in the DOM? Seems simpler.

Reply:
Because of the animations. If we keep it always in the dom, the "hidden" animation will always work and it will make a weird effect.

Reply:
Ah yes. Add a `transition` for `visibility` from `visible` to `hidden` over the same duration as the animation (we should have this end up with `visibility: hidden` anyway to avoid getting keyboard focus after it's gone).